src/components/ButtonPanel.js

```diff
@@ -1,38 +1,53 @@
 import React from 'react';
+import PropTypes from 'prop-types';
 import Button from './Button';
 
-const ButtonPanel = () => (
-  <div>
-    <div id="group1">
-      <Button name="AC" />
-      <Button name="+/-" />
-      <Button name="%" />
-      <Button name="÷" />
-    </div>
```

Comment:
**OPTIONAl**
We can store all the button names in an array and then create a list of buttons using the map callback function on the array in order to avoid code redundancy.
For example: 
For group1, we can have a constant containing button names for group1
```js
const group1 = ["AC", "+/-", "%", "/"] 
```
Then display them using a callback function on the array:
```js
{ group1.map(val => <Button name={val} />)}
```